data/common.yaml

```
@@ -0,0 +1,4 @@
+ssh::service: 'ssh'
+ssh::client_package: undef
+ssh::server_package: undef
```

undef should use tilder
```suggestion
ssh::server_package: ~
```

---

also we need to include all defaults

```yaml
ssh::manage_client: true
ssh::manage_server: true
ssh::server::challenge_response_authentication: false
ssh::server::root_authentication: false
ssh::server::password_authentication: true
ssh::server::x11_forwading: true
ssh::server::print_mod: false
ssh::server::package: ~
ssh::server::ensure: 'running'
ssh::server::enabled: true
ssh::client::check_host_ip: true
ssh::client::package: ~
```